doc/userguide/rules/integer-keywords.rst

```diff
@@ -142,4 +142,17 @@ For the array [1,2,3,4,5,6], here are some examples:
 * 3:-1 will have subslice [4,5]
 * -4:4 will have subslice [3,4]
 
-If one index is out of bounds, an empty subslice is used.
\ No newline at end of file
+If one index is out of bounds, an empty subslice is used.
+
+Count
+-----
+
+Multi-integer can also just count the number of occurences
+without matching to a specific value.
+
+The syntax is::
+ keyword: count [mode] value;
```

Comment:
Should we document the usages such as `all`, `all1`, `nb` etc that are also valid?

Reply:
> TODOs :
> 
> * update doc


That is what I meant

Unless you are referring to https://github.com/OISF/suricata/pull/13897 ?

Reply:
I had missed the `TODO` description in the PR, apologies.